src/js/components/header.tsx

```diff
@@ -1,23 +1,126 @@
+import Icon from '@salesforce/design-system-react/components/icon';
+import Dropdown from '@salesforce/design-system-react/components/menu-dropdown';
 import PageHeader from '@salesforce/design-system-react/components/page-header';
 import PageHeaderControl from '@salesforce/design-system-react/components/page-header/control';
-import React from 'react';
+import i18n from 'i18next';
+import React, { useCallback } from 'react';
 import { useSelector } from 'react-redux';
-import { Link } from 'react-router-dom';
+import {
+  Link,
+  match as Match,
+  useHistory,
+  useRouteMatch,
+} from 'react-router-dom';
 
+import backpackIcon from '~img/backpack-sm.svg';
+import mapIcon from '~img/map-sm.svg';
+import seesawIcon from '~img/seesaw-sm.svg';
 import Errors from '~js/components/apiErrors';
 import OfflineAlert from '~js/components/offlineAlert';
 import Toasts from '~js/components/toasts';
 import UserInfo from '~js/components/user/info';
+import { AppState } from '~js/store';
+import { selectProject } from '~js/store/projects/selectors';
 import { selectSocketState } from '~js/store/socket/selectors';
 import { selectUserState } from '~js/store/user/selectors';
-import routes from '~js/utils/routes';
+import {
+  SHOW_WALKTHROUGH,
+  WALKTHROUGH_TYPES,
+  WalkthroughType,
+} from '~js/utils/constants';
+import routes, { routePatterns } from '~js/utils/routes';
+
+const TourLabel = ({
+  data: { label, iconPath, iconName },
+}: {
+  data: {
+    label: string;
+    iconPath: string;
+    iconName: string;
+  };
+}) => (
+  <span className="slds-truncate" title={label}>
+    <Icon
```

Comment:
Probably not new in this branch, but it looks like icons are showing up as unlabeled `role=diagram` as the default role for svgs. We probably want to add `aria-hidden="true" focusable="false"` like we do on other sites, if that's an option.

Reply:
@mirisuzanne I might be misunderstanding, but from what I can tell all of our SVG icons have `aria-hidden="true"` as per SLDS specs. I think we can add `focusable="false"` if that's important.